src/Extensions/Compatability/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace Microsoft.Omex.Extensions.Compatability
+{
+	/// <summary>
+	/// Class to register dependencies for Compatability classes
+	/// </summary>
+	public static class ServiceCollectionExtensions
+	{
+		/// <summary>
+		/// Initialize deprecated static classes like Code and ULSLogger
+		/// </summary>
+		public static IHostBuilder AddOmexCompatabilityServices(this IHostBuilder builder) =>
```

Comment:
Validate this before use

Reply:
I don't want to overload code with more validation since it's quite an uncommon case when you will call extension on a non-nullable object that is null and even in this case user will get `ArgumentNullException` from `ConfigureServices` extension method that would be identical to the one I would throw.